lib/pana.dart

```diff
@@ -64,6 +66,18 @@ class PackageAnalyzer {
         upgrade.exitCode, upgrade.stdout, upgrade.stderr, pkgDir);
     log.info("Package version: ${summary.pkgVersion}");
 
+    LibraryScanner libraryScanner = new LibraryScanner(package, pkgDir);
+    Map<String, List<String>> directLibs;
+    Map<String, List<String>> transitiveLibs;
+    try {
+      directLibs = await libraryScanner.scanDirectLibs();
+      transitiveLibs = await libraryScanner.scanTransitiveLibs();
```

Comment:
Is one more likely to fail than the other? Any reason why these are not ran separately?